ChangeLog

```diff
@@ -7,6 +7,8 @@ Bugfix
      invalidated keys of a lifetime of less than a 1s. Fixes #1968.
    * Fix failure in hmac_drbg in the benchmark sample application, when
      MBEDTLS_THREADING_C is defined. Found by TrinityTonic, #1095
+   * Fix an unsafe bounds check in ssl_load_session(). Reported by
```

Comment:
I think "unsafe bounds check" is alarming enough that people might question why it's in the Bugfix section, not Security. We might want to make it clear that this is not exploitable without breaking the authentication of the session ticket.

Reply:
Fixed.